### ISSUE TITLE
Test in Firefox

### ISSUE THREAD

**Issue by pgte** [peer-base/peer-base#206]
Currently the tests run in Chrome only. Fix this.

**Comment by dirkmc** [comment]
I think in Travis they run in Firefox, and locally they run in Chrome:
https://github.com/ipfs/aegir/blob/master/src/config/karma.conf.js#L12

It seems like in CI we're only running browser tests, not nodejs tests, is that right?
https://github.com/peer-base/peer-base/blob/master/.travis.yml#L10

**Comment by pgte** [comment]
@dirkmc yes, it seams you're right. I think local tests should also run in Firefox, no'

**Comment by dirkmc** [comment]
That makes sense, I wonder why they did that in aegir
